tests/examples/minimal/pom.xml

```diff
@@ -39,7 +41,7 @@
         <dependency>
             <groupId>com.pulumi</groupId>
             <artifactId>pulumi</artifactId>
-            <version>${env.PULUMI_JAVA_SDK_VERSION}</version>
+            <version>${pulumiSdkVersion}</version>
```

Comment:
I'm a bit scared messing with these

Reply:
why, this is very easy, this is the canonical way of doing this version hack